docs/quickstart.md

```diff
@@ -194,9 +197,9 @@ When the `principals create` command completes successfully, it will return the
 {"clientId": "XXXX", "clientSecret": "YYYY"}
 ```
 
-Now, we grant the principal the [principal role](./entities.md#principal-role) we created, and grant the [catalog role](./entities.md#catalog-role) the principal role we created. For more information on these entities, please refer to the linked documentation.
+Now, we grant to the principal `quickstart_user` the [principal role](./entities.md#principal-role) `quickstart_user_role`; then to this principal role, we grant the [catalog role](./entities.md#catalog-role) `quickstart_catalog_role`. For more information on these entities, please refer to the linked documentation.
```

Comment:
This seems like a grammar/style edit and is the only change here that I think could be controversial. 

If you don't feel strongly about this line, would you be okay merging without this change? Otherwise, do you have a reason that you think this construction ("indirect object first") reads better?

Reply:
The main reason for inverting the indirect object position was to avoid the awkward "grant the principal the principal".

I'm fine removing this change though.